java/opendcs/src/main/java/opendcs/dai/DacqEventDAI.java

```diff
@@ -93,6 +95,22 @@ public int readEventsForScheduleStatus(DbKey scheduleEntryStatusId, ArrayList<Da
 	 */
 	public int readEventsForPlatform(DbKey platformId, ArrayList<DacqEvent> evtList)
 		throws DbIoException;
+
+	/**
+	 * Read events for a specific DACQ Event matching the specified criteria.
+	 * Events are added to the end of the passed array in time order.
+	 * If evtList is not empty, only events after the last in the array are read.
+	 * @param evtList the list to add events to
+	 * @param appId the application ID
+	 * @param routingExecId the routing execution ID
+	 * @param platformId the platform ID
+	 * @param backlog the backlog
+	 * @param httpSession the HTTP session
+	 * @return number of events added to the list
+	 * @throws DbIoException
+	 */
+	public int readEvents(ArrayList<DacqEvent> evtList, DbKey appId, DbKey routingExecId, DbKey platformId, String backlog, HttpSession httpSession)
```

Comment:
not that we can really get away with it but I'm okay with pretending this DAI doesn't actually exist.